web/src/util/cdc-loader.js

```diff
@@ -0,0 +1,5 @@
+import raw from "raw.macro"
+
+export const loadTransaction = () => {}
+
+export const loadScript = () => {}
```

Comment:
Oooh, this looks cool. Is this a WIP?

Reply:
I've been thinking more about how to bridge the gap between the `/cadence` folder and the JS code. I think it probably involves something like this.

Reply:
Yes exactly. I'll remove this, because I think it warrants its own PR, but the idea is to explore loading Cadence from local files, in a build-agnostic way.

Reply:
I'll rename this PR and create a new PR for that work.